### PR TITLE
[MB-1555] Add airship ready intent

### PR DESCRIPTION
**Pull request by rlepinski**
Need it for the gimbal bridge.

### CODE REVIEW

**Comment by marc-scig** [comment]
📥

**Comment by hcrowell** [comment]
🍊

**Comment by z0br0wn** [comment]
😎

File: src/android/CordovaAutopilot.java [hunk 1]
```diff
@@ -198,6 +201,13 @@ public void onInboxUpdated() {
             }
         });
 
+
+        // Send AirshipReady intent so other plugins that depend on Urban Airship can initailize
```
Comment (hcrowell) [inline src/android/CordovaAutopilot.java]: initialize*

**Pull request merged**